cmd/vc-rest/startcmd/params.go

```diff
@@ -204,6 +204,18 @@ const (
 	requestObjectRepositoryS3RegionEnvKey    = "REQUEST_OBJECT_REPOSITORY_S3_REGION"
 	requestObjectRepositoryS3RegionFlagUsage = "request-object S3 Region"
 
+	cslStoreTypeFlagName = "csl-store-type"
+	cslStoreTypeEnvKey   = "CSL_STORE_TYPE"
+	cslStoreFlagUsage    = "Store type for CSL (Credential Status List). Supported: mongodb,s3. Default: mongodb"
+
+	cslStoreS3BucketFlagName  = "csl-store-s3-bucket"
+	cslStoreS3BucketEnvKey    = "CSL_STORE_S3_BUCKET"
+	cslStoreS3BucketFlagUsage = "CSL (Credential Status List) S3 Bucket"
+
+	cslStoreS3RegionFlagName  = "csl-store-s3-region"
+	cslStoreS3RegionEnvKey    = "CSL_STORE_S3_REGION"
+	cslStoreS3RegionFlagUsage = "CSL (Credential Status List) S3 Region"
+
```

Comment:
You need to add one more var s3 hostname like this https://github.com/trustbloc/vcs/commit/a5ef9734d138d06aabbe9a7a96a8f612fa489775

Reply:
sure, done